frontend/lib/map/map/map.dart

```diff
@@ -65,9 +81,14 @@ class _MapState extends State<Map> {
         if (widget.onFeatureClick != null) widget.onFeatureClick(featureInfo);
         var coordinates = featureInfo["geometry"]["coordinates"];
         var location = new LatLng(coordinates[1], coordinates[0]);
+        await _removeSymbol();
+        await _addSymbol(location, featureInfo["properties"]["categoryId"]);
         await _bringCameraToLocation(location);
       }
-    } else if (widget.onNoFeatureClick != null) widget.onNoFeatureClick();
+    } else if (widget.onNoFeatureClick != null) {
+      await _removeSymbol();
+      widget.onNoFeatureClick();
+    }
```

Comment:
Actually, I only did this, because I was curious how we could actually implement this.
Unfortunately, I couldn't find any other solution than drawing a symbol on top of the actual symbol.
Do you know of any other way?